### ISSUE TITLE
Dataset processing

### ISSUE THREAD

**Issue by maherr13** [ShenhanQian/SpeechDrivesTemplates#10]
I have a few questions regarding the dataset processing pipeline, 

 - at generate_clips script, why the start index is 80 ?? 
 - why there are 13  records in each clip labeled idle in the train test split file ?? 
 - are there any parameters I would need to adjust when creating my own data set??

btw there is an error in `3_2_split_train_val_test.py` that you naming the validation samples "val" while the model searches for "dev" labeled records.

**Comment by Garfield-Finch** [comment]
Thank you for your interest in our work.

1. In our dataset, there is always a segment of introduction that we want to desert. We will change that part to be a parameter user can set. Thank you for that.
2. It is because we want to make sure that the training set and test set do not overlap. 
3. As far as we are concerned, all the parameters that should be set by yourself have been indicated as "required=True" in the "argparser". It should be fine if you go by our default setting. If you find anything that is crucial but we did not notice, please follow up on this issue.

Thank you. That is an incompatibility between our naming rules and that of the original dataset from "Speech2Gesture"

**Comment by maherr13** [comment]
Thank you for the illustration.

I have a question regarding the data collection, from your experiments How much data do I need to collect (in hours) to get good results in general, and as good as Oliver specifically?

**Comment by ShenhanQian** [comment]
Here are the lengths of our training sequences for your reference:
| Subject| Length (hours)|
| :-:| :-:|
| Oliver| 11|
| Kubinec| 3|
| Luo| 7|
| Xing| 2|

Besides the length, the variation and the quality of pose may also highly influence the results. So we suggest collecting videos with expressive gestures like Oliver's and visually checking if the detected keypoints are correct and stable.

**Comment by maherr13** [comment]
Many Thanks. @ShenhanQian @Garfield-Finch
